src/components/Template/_Template.scss

```diff
@@ -1,5 +1,5 @@
 $sidebar-width: 25%;
-$sidebar-width-s: calc(25% - calc(4 * #{var(--space-s)}));
+$sidebar-width-s: calc(25% - calc(4 * var(--space-s)));
```

Comment:
The css variable doesn't need to be interpolated. I originally thought it was because this is doing `25% - (4 x 16px)` and the units were not the same. It doesn't seem to care and the UI looks the same in Storybook. But can this be double-checked, please?

Reply:
You got it—this is cruft from I assume Sass needing to interpolate different values but the native browser calc _not_ needing to interpolate values